frontend/appflowy_flutter/packages/appflowy_editor/test/service/shortcut_event/shortcut_event_test.dart

```diff
@@ -219,5 +219,42 @@ void main() async {
         Selection.single(path: [1], startOffset: text.length),
       );
     });
+
+    //TODO(JRS296): How to test Italicized state of textnode
```

Comment:
You can use this extension function.

```
  bool allSatisfyItalicInSelection(Selection selection) =>
      allSatisfyInSelection(selection, BuiltInAttributeKey.italic, (value) {
        return value == true;
      });
```

Reply:
Alright, will look into it 👍

Reply:
Just check if the `textNode.attributes[BuiltInAttributeKey.italic]` is true.